packages/dcos-integration-test/extra/resiliency/test_aws_cf.py

```diff
@@ -0,0 +1,75 @@
+import logging
```

Comment:
Alternatively, we could also just use a fixture that skips if some env var is set which sets something in Cluster, which doesnt seem much different, but its how we mark auth tests for skipping security disabled tests